cmd/create/cluster/cmd.go

```diff
@@ -803,6 +803,9 @@ func run(cmd *cobra.Command, _ []string) {
 	if isHostedCP && args.ec2MetadataHttpTokens != "" {
 		r.Reporter.Errorf("ec2-metadata-http-tokens can't be set with hosted-cp")
 		os.Exit(1)
+	} else if !isHostedCP && args.ec2MetadataHttpTokens == "" {
```

Comment:
NP
We can simplify the condition:
```suggestion
	if !isHostedCP && args.ec2MetadataHttpTokens == "" {
```
Do we need the `else` here? if the previous condition didn't apply we know that:
`isHostedCp == false` or `args.ec2MetadataHttpTokens == ""`